File: 0268-missing-number/solution.java
class Solution {
    public int missingNumber(int[] nums) {
        int sum_N=(nums.length*(nums.length+1))/2;
        int sum_arr=0;
        for(int i=0;i<nums.length;i++){
            sum_arr=sum_arr+nums[i];
        }
        int diff=sum_N-sum_arr;
        return diff;
    }
}

File: 0268-missing-number/solution.py
class Solution:
    def missingNumber(self, nums: List[int]) -> int:
        n=len(nums)
        sumnum=(n*(n+1))/2
        sum1=0
        for i in nums:
            sum1=sum1+i
        diff=sumnum-sum1
        return int(diff)
